yt/geometry/coordinates/coordinate_handler.py

```diff
@@ -70,38 +71,46 @@ def __init__(self, ds, ordering):
 
     def setup_fields(self):
         # This should return field definitions for x, y, z, r, theta, phi
-        raise NotImplementedError
+        pass
```

Comment:
Why isn't this one not an abstract method?

Reply:
looks like a mistake on my part, thanks for catching it !